test/models/test_quantized_model_for_pixart.py

```diff
@@ -0,0 +1,86 @@
+import os
+from tempfile import TemporaryDirectory
+
+import pytest
+import torch
+
+from optimum.quanto import QModuleMixin, is_diffusers_available, qint4, qint8
+
+
+def quantized_model_for_pixart(qtype, exclude):
+    from diffusers import PixArtTransformer2DModel
```

Comment:
Having these imports here is required to avoid import errors when diffusers is not installed.

Reply:
Yeah.